autoload/clap/installer.vim

```diff
@@ -7,9 +7,11 @@ set cpoptions&vim
 let s:plugin_root_dir = fnamemodify(g:clap#autoload_dir, ':h')
 
 function! s:run_term(cmd, cwd, success_info) abort
-  belowright 10new bottom
+  belowright 10new
```

Comment:
When user enabled swapfile, this line will have an error, saying trying to rename a swap file. So I just removed  the name of this window.

Reply:
Ok to me.